From 2c9bf2b6a7c1dc0e0b87e8974bc10d1da224e613 Mon Sep 17 00:00:00 2001
From: Matt Selsky <matthew.selsky@twosigma.com>
Date: Fri, 9 Dec 2016 00:17:20 -0500
Subject: [PATCH 243/268] Now that waf is fixed, revert minimum Python 2.6 back
 to 2.6.0

Reverts "Change pre-requisites to indicate Python 2 must be at 2.6.6 or later."
---
 INSTALL           | 2 +-
 devel/hacking.txt | 2 +-
 pylib/wscript     | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/INSTALL b/INSTALL
index 9c9048a..58ebfc3 100644
--- a/INSTALL
+++ b/INSTALL
@@ -12,7 +12,7 @@ via getifaddrs(3) or some equivalent facility.
 There are some prerequisites.  Libraries need the binary installed
 to run and in addition, the development headers installed to build.
 
-Python 2.x.y, x.y >= 6.6, or Python 3.x, x >= 3::
+Python 2.x, x >= 6, or Python 3.x, x >= 3::
    Required to build, and for various scripts such as ntpviz (but see
    the guidance for packagers in devel/packaging.txt).  Our Python code
    has been written polyglot to also run with production versions of
diff --git a/devel/hacking.txt b/devel/hacking.txt
index e269e23..437ccf8 100644
--- a/devel/hacking.txt
+++ b/devel/hacking.txt
@@ -177,7 +177,7 @@ the compiler and toolchain have been modernized.
 
 == Python guidelines ==
 
-You may assume Python 2 at 2.6.6 or later, or Python 3 at 3.3 or later.
+You may assume Python 2 at 2.6 or later, or Python 3 at 3.3 or later.
 
 Please read https://www.python.org/dev/peps/pep-0008/[PEP 8] and use
 that style.  The only PEP 8 style rule we relax is that you may
diff --git a/pylib/wscript b/pylib/wscript
index af72f21..d469c84 100644
--- a/pylib/wscript
+++ b/pylib/wscript
@@ -3,7 +3,7 @@ def options(opt):
 
 def configure(conf):
     conf.load('python')
-    conf.check_python_version((2, 6, 6))
+    conf.check_python_version((2, 6, 0))
     conf.check_python_headers()
 
 def build(ctx):
-- 
2.7.4

